### ISSUE TITLE
Accessibility Issue - Service ID Read Incorrectly by Screen Reader

### ISSUE THREAD

**Issue by sherenemorris03** [DFE-Digital/find-information-products-services-frontend#118]
**Describe the bug**
Issue 10 in the Accessibility Report
The table data for the ID of the service includes additional information for screen reader users. However, the content does not have a space between “Unique” and “ID.” This causes the screen reader to read the data as one word, which does not make sense and may confuse users relying on assistive technology.

User story:
As a screen reader user,
I want the service ID to be read clearly and correctly,
So that I can understand the information without confusion.

**Steps To reproduce**
1. Navigate to the Product description page.
2. Use a screen reader to read the Service ID cell.
3. Observe that “UniqueID” is read as a single word.

**Expected behaviour**
Screen readers should read “Unique ID” as two distinct words.
Screen readers read “UniqueID” as one word, causing confusion.

**Screenshots**

<img width="397" height="280" alt="Image" src="https://github.com/user-attachments/assets/eec89fc8-8916-4369-8889-fe8e82cb3bd6" />

<img width="704" height="266" alt="Image" src="https://github.com/user-attachments/assets/4845175d-1fb1-4ab2-b8cc-dbc884b8f137" />

**Additional context**
Remove the space between Unique and ID which will resolve the issue.

Link to issue on Statement Service
https://accessibility-statements.education.gov.uk/services/DACFEE24-0602-46AD-AE7C-19A23DADA42D/issues/87B374F9-FE19-471E-B91D-33225663D24E
